war/src/main/less/abstracts/theme.less

```diff
@@ -151,23 +160,38 @@
   --link-dark-font-weight: 600;
 
   // Pane
-  --pane-header-color: #eee;
-  --pane-header-border-color: #f3f3f3;
-  --pane-border-color: var(--very-light-grey);
+  --pane-border-width: 1px;
+  --pane-header-text-color: var(--white);
+  --pane-header-bg: var(--dark-grey);
+  --pane-header-border-color: var(--light-grey);
+  --pane-header-font-weight: bold;
+  --pane-border-color: var(--light-grey);
   --pane-text-color: #3b3b3b;
   --pane-link-color: black;
   --pane-link-color--visited: black;
 
-  // TabBar
-  --tab-bar-border-color: var(--very-light-grey);
-  --tab-bar-text-color: #999;
-  --tab-bar-bg--checked: #eee;
-  --tab-bar-border-color--checked: var(--very-light-grey);
-  --tab-bar-text-color--checked: #000;
-  --tab-bar-bg-color--hover: #eee;
-  --tab-bar-add-text-color: #0460d1;
-  --tab-bar-add-bg-color--hover: #d8efff;
-  --tab-bar-add-border-color: #d8efff;
+  // TabBar links
+  --tab-link-border-radius: 4px;
+  --tab-link-border-width: 2px;
+  --tab-link-padding-x: 0.75rem;
+  --tab-link-padding-y: 0.375rem;
+  --tab-link-font-weight: bold;
+  --tab-link-text-color: var(--text-color-secondary);
+  --tab-link-bg: transparent;
+  --tab-link-border-color: var(--light-grey);
+  --tab-link-text-color--hover: var(--tab-link-text-color); // Redundant?
+  --tab-link-bg--hover: var(--light-grey);
+  --tab-link-border-color--hover: var(--tab-link-bg--hover); // Redundant?
+  --tab-link-text-color--active: var(--tab-link-text-color); // Redundant?
+  --tab-link-bg--active: var(--light-grey);
+  --tab-link-border-color--active: var(--tab-link-checked-bg); // Redundant?
```

Comment:
@timja I think I might have gone too hard on the CSS var API. WDYT?

Reply:
it's a lot, but seems quite good.